pkg/controller/kataconfig/openshift_reconciler.go

```diff
@@ -162,6 +162,7 @@ func (r *ReconcileKataConfigOpenShift) isOldestCR() (bool, error) {
 
 func (r *ReconcileKataConfigOpenShift) processDaemonsetForCR(operation DaemonOperation) *appsv1.DaemonSet {
 	runPrivileged := true
+	configmapOptional := true
```

Comment:
maybe group all the vars under a var block: 

example: 
```go
var (
	runPrivileged = true
	configmapOptional = true
	var runAsUser int64 = 0
)
```

Reply:
:+1: